pkg/epp/scheduling/types/types.go

```diff
@@ -78,7 +78,13 @@ func ToSchedulerPodMetrics(pods []backendmetrics.PodMetrics) []Pod {
 	return pm
 }
 
-// Result captures the scheduler result.
-type Result struct {
+// ProfileRunResult captures the profile run result.
+type ProfileRunResult struct {
 	TargetPod Pod
```

Comment:
this is supposed to be a list, or is that planned in a followup?

Reply:
yes. it should be a list but is not addressed in this PR which focused on ProfileHandler changes and adding the ProcessResult extension point.
will be addressed in a follow up.